app/services/hasura_service.rb

```diff
@@ -0,0 +1,43 @@
+class HasuraService
```

Comment:
Would a better name be `HasuraJwtService`?
Thinking in case we have other future needs for Hasura type things... hopefully not!

Reply:
I was thinking those other things can also go into that service, basically everything related to Hasura goes here 🤔